MobileCenterCrashes/MobileCenterCrashes/Internals/MSWrapperCrashesHelper.h

```diff
@@ -17,4 +20,24 @@
  */
 + (id<MSCrashHandlerSetupDelegate>)getCrashHandlerSetupDelegate;
 
+/**
+ * Disables automatic crash processing. Causes SDK not to send reports, even if ALWAYS_SEND is set.
```

Comment:
`setAutomaticProcessing` is a setter method, the comment for a setter method should say something like "set XXX value" instead of "disable XXX value". This current comment is confusing business logic of the code with programming interfaces. We should move this Line24 comment to the RN SDK calling method.

Reply:
It's actually not a setter method. The property doesn't exist in this class. It exists in the crashes class.